# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/webaudio/MediaStreamAudioDestinationNode.cpp
# Commit: dea9f12cdec0
# Full Hash: dea9f12cdec02aeb5ac260a970659e09dc73a375
# Author: Andreas Pehrson <apehrson@mozilla.com>
# Date: 2025-07-02 09:15:46
# Regressor Bug: 1560936
# File Overlap Count: 3
# Description:
#   Bug 1560936 - For getSettings make all MediaStreamTrackSources populate width and height as appropriate. r=jib,bwc
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D251430
# ==============================================================================

diff -r a4ae1edb514a -r dea9f12cdec0 dom/media/webaudio/MediaStreamAudioDestinationNode.cpp
--- a/dom/media/webaudio/MediaStreamAudioDestinationNode.cpp	Tue Jul 01 18:42:42 2025 +0000
+++ b/dom/media/webaudio/MediaStreamAudioDestinationNode.cpp	Tue Jul 01 18:42:43 2025 +0000
@@ -49,6 +49,8 @@
     return MediaSourceEnum::AudioCapture;
   }
 
+  void GetSettings(MediaTrackSettings& aSettings) override {}
+
   void Stop() override { Destroy(); }
 
   void Disable() override {}